pyproject.toml

```diff
@@ -0,0 +1,66 @@
+[build-system]
+build-backend = "hatchling.build"
+requires = ["hatchling", "hatch-vcs"]
+
+[tool.hatch.build.targets.wheel]
+packages = ["nidmresults"]
+
+# [tool.hatch.build.hooks.vcs]
+# version-file = "neuropower/_version.py"
+
+# [tool.hatch.version]
+# source = "vcs"
```

Comment:
Cannot yet be propelerly used because the repo does not have the tags of the versions

Reply:
fixed